### PR TITLE
New package: i2pd-2.8.0

### PR DESCRIPTION
**Pull request by Kayssyn**

### CODE REVIEW

File: srcpkgs/i2pd/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+revision=1
+build_style=cmake
+short_desc="Full-featured C++ implementation of I2P client"
+maintainer="Kayssyn <kayssyn@gmail.com>"
+license="BSD"
+homepage="https://github.com/PurpleI2P/i2pd"
+build_options="upnp aesni"
+desc_option_upnp="build with UPnP support"
+desc_option_aesni="build with AES-NI support"
+hostmakedepends="cmake"
```
Comment (Vaelatern) [inline srcpkgs/i2pd/template]: Line not necessary

File: srcpkgs/i2pd/template [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+makedepends="
+	$(vopt_if upnp miniupnpc-devel)
+	boost-devel
```
Comment (Vaelatern) [inline srcpkgs/i2pd/template]: These can all go on one line please.

File: srcpkgs/i2pd/template [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	zlib-devel
+	libressl-devel"
+depends="
+	$(vopt_if upnp miniupnpc)
+	zlib
+	libressl"
```
Comment (Vaelatern) [inline srcpkgs/i2pd/template]: No need to depend on libressl

File: srcpkgs/i2pd/template [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+pre_configure() {
+	vlicense ${wrksrc}/LICENSE
```
Comment (Vaelatern) [inline srcpkgs/i2pd/template]: `vlicense LICENSE` and please add a line of whitespace between functions.

File: srcpkgs/i2pd/template [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+makedepends="
+	$(vopt_if upnp miniupnpc-devel)
+	boost-devel
+	zlib-devel
+	libressl-devel"
+depends="
+	$(vopt_if upnp miniupnpc)
+	zlib
+	libressl"
+pre_configure() {
```
Comment (Vaelatern) [inline srcpkgs/i2pd/template]: Functions below other variables being set, as a general rule.

File: srcpkgs/i2pd/template [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+	vlicense ${wrksrc}/LICENSE
+}
+build_wrksrc="build"
+configure_args="
+	-DCMAKE_BUILD_TYPE=Release
```
Comment (Vaelatern) [inline srcpkgs/i2pd/template]: Again, all one line.

File: srcpkgs/i2pd/template [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# Template file for 'i2pd'
+pkgname=i2pd
+version=2.8.0
+revision=1
+build_style=cmake
+short_desc="Full-featured C++ implementation of I2P client"
+maintainer="Kayssyn <kayssyn@gmail.com>"
+license="BSD"
+homepage="https://github.com/PurpleI2P/i2pd"
+build_options="upnp aesni"
```
Comment (Vaelatern) [inline srcpkgs/i2pd/template]: Generally the build options go after the rest of the variables, before the functions.

**Comment by Vaelatern** [comment]
Nice work so far.

File: srcpkgs/i2pd/template (hunk 4 above)
Comment (Kayssyn) [inline srcpkgs/i2pd/template]: `vlicense LICENSE` fails:

```
=> i2pd-2.8.0_1: running pre_configure ...
=> ERROR: i2pd-2.8.0_1: vinstall: cannot find 'LICENSE'...
=> ERROR: i2pd-2.8.0_1: failed to run pre_configure() at line 140.
```
Maybe it tries to find LICENSE in build directory?

**Comment by Kayssyn** [comment]
Fixed all except LICENSE path.

File: srcpkgs/i2pd/template (hunk 4 above)
Comment (Kayssyn) [inline srcpkgs/i2pd/template]: Changed it to `post_install()`

**Comment by Kayssyn** [comment]
Sorry, tried to commit new version, but it doesn't compiles at all. Also had some problems with git reversing that commit

**Pull request closed without merging**